include/xtl/xsequence.hpp

```diff
@@ -200,6 +200,16 @@ namespace xtl
     {
         return N;
     }
+
+    /************************
+     * equivalent_sequences *
+     ************************/
+
+    template <class E1, class E2>
+    inline bool equivalent_sequences(const E1& e1, const E2& e2)
```

Comment:
Should we name this method `are_equivalent_sequences` to be consistent with the `is_XXX` naming convention for functions that return boolean values?

Reply:
Good idea